[Bug] Docker file needs update of a cmake flag

The current docker file seems to be missing out a `DCMAKE_INSTALL_PREFIX` flag. in one of the `cmake` command. Because of which the following error occurs:

```bash
-- Generating done
-- Build files have been written to: /nvstrings/python/build/temp.linux-x86_64-3.7
Scanning dependencies of target pyniNVStrings
[ 50%] Building CXX object CMakeFiles/pyniNVStrings.dir/pystrings.cpp.o
/nvstrings/python/cpp/pystrings.cpp: In function 'PyObject* n_createFromOffsets(PyObject*, PyObject*)':
/nvstrings/python/cpp/pystrings.cpp:436:64: error: no matching function for call to 'NVStrings::create_from_offsets(const char*&, int&, const int*&, const unsigned char*&, int&, bool&)'
                                          nbuffer,ncount,bdevmem);
                                                                ^
In file included from /nvstrings/python/cpp/pystrings.cpp:25:0:
/conda/envs/cudf/include/nvstrings/NVStrings.h:115:23: note: candidate: static NVStrings* NVStrings::create_from_offsets(const char*, int, const int*, const unsigned char*, int)
     static NVStrings* create_from_offsets(const char* strs, int count, const int* offsets, const unsigned char* nullbitmask=0, int nulls=0);
                       ^
/conda/envs/cudf/include/nvstrings/NVStrings.h:115:23: note:   candidate expects 5 arguments, 6 provided
/nvstrings/python/cpp/pystrings.cpp: In function 'PyObject* n_cat(PyObject*, PyObject*)':
/nvstrings/python/cpp/pystrings.cpp:1505:41: error: no matching function for call to 'NVStrings::cat(std::vector<NVStrings*>&, const char*&, const char*&)'
         rtn = tptr->cat(others,sep,narep);
                                         ^
In file included from /nvstrings/python/cpp/pystrings.cpp:25:0:
/conda/envs/cudf/include/nvstrings/NVStrings.h:412:16: note: candidate: NVStrings* NVStrings::cat(NVStrings*, const char*, const char*)
     NVStrings* cat( NVStrings* others, const char* separator, const char* narep=0);
                ^
/conda/envs/cudf/include/nvstrings/NVStrings.h:412:16: note:   no known conversion for argument 1 from 'std::vector<NVStrings*>' to 'NVStrings*'
CMakeFiles/pyniNVStrings.dir/build.make:62: recipe for target 'CMakeFiles/pyniNVStrings.dir/pystrings.cpp.o' failed
make[3]: *** [CMakeFiles/pyniNVStrings.dir/pystrings.cpp.o] Error 1
CMakeFiles/Makefile2:109: recipe for target 'CMakeFiles/pyniNVStrings.dir/all' failed
make[2]: *** [CMakeFiles/pyniNVStrings.dir/all] Error 2
CMakeFiles/Makefile2:121: recipe for target 'CMakeFiles/pyniNVStrings.dir/rule' failed
make[1]: *** [CMakeFiles/pyniNVStrings.dir/rule] Error 2
Makefile:131: recipe for target 'pyniNVStrings' failed
make: *** [pyniNVStrings] Error 2
Traceback (most recent call last):
  File "setup.py", line 36, in <module>
    zip_safe=False
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "/conda/envs/cudf/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/conda/envs/cudf/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/conda/envs/cudf/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/install.py", line 109, in do_egg_install
    self.run_command('bdist_egg')
  File "/conda/envs/cudf/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/conda/envs/cudf/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 172, in run
    cmd = self.call_command('install_lib', warn_dir=0)
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/bdist_egg.py", line 158, in call_command
    self.run_command(cmdname)
  File "/conda/envs/cudf/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/conda/envs/cudf/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/install_lib.py", line 11, in run
    self.build()
  File "/conda/envs/cudf/lib/python3.7/distutils/command/install_lib.py", line 107, in build
    self.run_command('build_ext')
  File "/conda/envs/cudf/lib/python3.7/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/conda/envs/cudf/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/conda/envs/cudf/lib/python3.7/site-packages/setuptools/command/build_ext.py", line 78, in run
    _build_ext.run(self)
  File "/conda/envs/cudf/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 186, in run
    _build_ext.build_ext.run(self)
  File "/conda/envs/cudf/lib/python3.7/distutils/command/build_ext.py", line 340, in run
    self.build_extensions()
  File "/conda/envs/cudf/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 194, in build_extensions
    self.build_extension(ext)
  File "/conda/envs/cudf/lib/python3.7/site-packages/cmake_setuptools/cmake/__init__.py", line 59, in build_extension
    env=env)
  File "/conda/envs/cudf/lib/python3.7/subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-j', 'pyniNVStrings']' returned non-zero exit status 2.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -27,7 +27,7 @@
 - PR #335 to_host no longer overwrites input pointers for null strings
 - PR #329 Fixed documentation errors in source code
 - PR #334 Regex stack-size logic moved to host code
-
+- PR #343 Fixed a flag which is necessary for conda install in Dockerfile
 
 # cuStrings/nvStrings 0.7.0 (10 May 2019)
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,8 +11,6 @@ ADD thirdparty /nvstrings/thirdparty
 WORKDIR /nvstrings/cpp
 
 ENV CONDA_ENV=cudf
-ENV PYNI_PATH=/conda/envs/${CONDA_ENV}
-ENV PYTHON_VERSION=3.7
 
 RUN locale-gen en_US.UTF-8  
 ENV LANG en_US.UTF-8  
@@ -22,6 +20,7 @@ ENV LC_ALL en_US.UTF-8
 # Remove the cudf installed nvstrings
 RUN source activate ${CONDA_ENV} && \
     conda remove nvstrings && \
+    conda remove libnvstrings && \
     pip install cmake_setuptools
 
 # Build

--- a/Dockerfile
+++ b/Dockerfile
@@ -27,7 +27,7 @@ RUN source activate ${CONDA_ENV} && \
 # Build
 RUN source activate ${CONDA_ENV} && \
     mkdir build && cd build && \
-    cmake .. && make -j install
+    cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} && make -j install
 
 # Install
 WORKDIR /nvstrings/python

